<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainPage</class>
 <widget class="QWidget" name="mainPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>902</width>
    <height>519</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Основные настройки</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="6" column="1">
    <widget class="QCheckBox" name="chkHighlightInvalidDate">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Подсвечивать даты с ошибками</string>
     </property>
     <property name="buddy">
      <cstring>chkHighlightInvalidDate</cstring>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QCheckBox" name="chkPersonFilterInCmb">
     <property name="text">
      <string/>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="10" column="2" colspan="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="10" column="0">
    <widget class="QLabel" name="lblPersonnelByDblClick">
     <property name="text">
      <string>Открывать список сотрудников по двойному щелчку</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0">
    <widget class="QLabel" name="lblTemplateDir">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Каталог с шаблонами печати документов</string>
     </property>
     <property name="buddy">
      <cstring>edtTemplateDir</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="4">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="5" column="2" colspan="2">
    <spacer name="horizontalSpacer_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>168</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="CPolyclinicComboBox" name="cmbOrganisation"/>
   </item>
   <item row="15" column="0">
    <widget class="QLabel" name="lblFTPUrl">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>URL Адрес FTP-cервера автообновления</string>
     </property>
     <property name="buddy">
      <cstring>edtFTPUrl</cstring>
     </property>
    </widget>
   </item>
   <item row="11" column="0" colspan="4">
    <widget class="Line" name="line_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="18" column="0">
    <spacer name="spacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>61</width>
       <height>149</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblDefaultCity">
     <property name="minimumSize">
      <size>
       <width>130</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>&amp;Нас.пункт по умолчанию</string>
     </property>
     <property name="buddy">
      <cstring>cmbDefaultCity</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblDefaultProvince">
     <property name="minimumSize">
      <size>
       <width>130</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>&amp;Область</string>
     </property>
     <property name="buddy">
      <cstring>cmbProvince</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblOrgStructure">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>130</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>&amp;Подразделение</string>
     </property>
     <property name="buddy">
      <cstring>cmbOrgStructure</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="CKLADRComboBox" name="cmbDefaultCity"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="lblOrganisation">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Maximum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>130</width>
       <height>0</height>
      </size>
     </property>
     <property name="text">
      <string>&amp;ЛПУ</string>
     </property>
     <property name="buddy">
      <cstring>cmbOrganisation</cstring>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="lblCmbPersonFilter">
     <property name="toolTip">
      <string>Возможность прямого ввода только тех врачей, которые отфильтрованы в выпадающем диалоге.</string>
     </property>
     <property name="text">
      <string>Ограничивать непосредственный выбор сотрудника по подразделению</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="4">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="15" column="1" colspan="2">
    <widget class="QLineEdit" name="edtFTPUrl"/>
   </item>
   <item row="0" column="3">
    <widget class="QToolButton" name="btnSelectOrganisation">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="COrgStructureComboBox" name="cmbOrgStructure"/>
   </item>
   <item row="8" column="2" colspan="2">
    <spacer name="horizontalSpacer_15">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>168</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="2" colspan="2">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="lblOnlyDoctors">
     <property name="text">
      <string>Выбор сотрудников только с врачебными специальностями</string>
     </property>
    </widget>
   </item>
   <item row="14" column="0" colspan="4">
    <widget class="Line" name="line_4">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="6" column="2" colspan="2">
    <spacer name="horizontalSpacer_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>168</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="1">
    <widget class="QCheckBox" name="chkHighlightRedDate">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="12" column="1" colspan="2">
    <widget class="QLineEdit" name="edtTemplateDir"/>
   </item>
   <item row="3" column="1" colspan="3">
    <widget class="CKLADRComboBox" name="cmbProvince"/>
   </item>
   <item row="8" column="1">
    <widget class="QCheckBox" name="chkOnlyDoctors">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Подсвечивать выходные и праздничные даты</string>
     </property>
     <property name="buddy">
      <cstring>chkHighlightRedDate</cstring>
     </property>
    </widget>
   </item>
   <item row="12" column="3">
    <widget class="QToolButton" name="btnSelectTemplateDir">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QCheckBox" name="chkPersonnelByDblClick">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="16" column="0" colspan="4">
    <widget class="Line" name="line_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="17" column="1" colspan="2">
    <widget class="QLineEdit" name="edtBrowserDir"/>
   </item>
   <item row="17" column="0">
    <widget class="QLabel" name="lblPathBrowser">
     <property name="text">
      <string>Адрес браузера в системе</string>
     </property>
    </widget>
   </item>
   <item row="17" column="3">
    <widget class="QToolButton" name="btnSelectBrowserDir">
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CPolyclinicComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">Orgs/OrgComboBox.h</header>
  </customwidget>
  <customwidget>
   <class>COrgStructureComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">Orgs/OrgStructComboBoxes.h</header>
  </customwidget>
  <customwidget>
   <class>CKLADRComboBox</class>
   <extends>QComboBox</extends>
   <header location="global">KLADR/kladrComboxes.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>cmbOrganisation</tabstop>
  <tabstop>btnSelectOrganisation</tabstop>
  <tabstop>cmbOrgStructure</tabstop>
  <tabstop>cmbDefaultCity</tabstop>
  <tabstop>cmbProvince</tabstop>
  <tabstop>chkHighlightRedDate</tabstop>
  <tabstop>chkHighlightInvalidDate</tabstop>
  <tabstop>chkOnlyDoctors</tabstop>
  <tabstop>chkPersonFilterInCmb</tabstop>
  <tabstop>chkPersonnelByDblClick</tabstop>
  <tabstop>edtTemplateDir</tabstop>
  <tabstop>btnSelectTemplateDir</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
